api/views/CloudRecordSummaryView.py

```diff
@@ -193,15 +193,17 @@ def _paginate_result(self, request, result):
         return serializer.data
 
     def _filter_cursor(self, cursor):
-        """Filter database results based on setting.RETURN_HEADERS."""
-        columns = cursor.description
-        results = []
-        for value in cursor.fetchall():
+        """Filter database results based on settings.RETURN_HEADERS."""
+        results = []  # Return list.
+        for record in cursor.fetchall():  # record refers to one days summary
+            # Construct a new summary with only the values we care about.
             result = {}
```

Comment:
You have `result` and `results` which are quite similar. Need to distinguish better.